### PR TITLE
Feat/725 replace application info page

### PR DESCRIPTION
**Pull request by StephGit**

### CODE REVIEW

File: AMW_angular/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (yvespp) [inline AMW_angular/package-lock.json:1]: This file is probably not needed

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (yvespp) [inline package-lock.json:1]: This file is probably not needed

File: AMW_angular/io/src/app/settings/settings.component.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
         <a class="nav-link" href="/AMW_web/pages/settings.xhtml?mode=STP">STP Management</a>
         <a class="nav-link" href="/AMW_web/pages/settings.xhtml?mode=releases">Releases</a>
         <a class="nav-link" routerLink="/settings/permission" routerLinkActive="active">Roles and Permissions</a>
-        <a class="nav-link" href="/AMW_web/pages/settings.xhtml?mode=appVersion">Application Info</a>
+        <a class="nav-link" routerLink="/settings/application-info" routerLinkActive="active">Application Info</a>
       </nav>
     </div>
     <div class="col-9">
```
Comment (yvespp) [inline AMW_angular/io/src/app/settings/settings.component.html:18]: Can we go to `col-10` here? With `cole-9` the page has unused extra space on the right:
image
This page could benefit from some more space and this size is also in line with the other pages (deploy and audit).

**Review by yvespp — COMMENTED** [review]
Cool, looks good otherwise!

File: AMW_angular/io/src/app/settings/settings.component.html (hunk 3 above)
Comment (StephGit) [reply AMW_angular/io/src/app/settings/settings.component.html]: yes, makes sense

**Review by yvespp — APPROVED** [review]

**Pull request merged**